### PR TITLE
TFM and SourceLink Changes

### PR DESCRIPTION
**Pull request by chidozieononiwu**
- Generate Target Moniker file in intermediate output location
- Update SourceLink to release version

### CODE REVIEW

File: eng/Directory.Build.Data.props [hunk 1]
```diff
@@ -61,6 +61,8 @@
 
     <RequiredTargetFrameworks>net452;netstandard1.4;net461;netstandard2.0</RequiredTargetFrameworks>
     <RequiredTargetFrameworks Condition="'$(SupportsNetStandard20)' == 'true'">netstandard2.0</RequiredTargetFrameworks>
+    
+    <TargetFrameworkMonikerAssemblyAttributesPath>$(IntermediateOutputPath)$(TargetFrameworkMoniker).AssemblyAttributes$(DefaultLanguageSourceExtension)</TargetFrameworkMonikerAssemblyAttributesPath>
```
Comment (weshaggard) [inline eng/Directory.Build.Data.props]: Is TargetFrameworkMoniker set at this point?

Comment (chidozieononiwu) [reply eng/Directory.Build.Data.props]: Looking it the log it is set and used several times before this is called.

Comment (weshaggard) [reply eng/Directory.Build.Data.props]: We should simply double check to make sure it is set as expected for projects that have more then one target framework. Generally the TargetFrameworkMoniker is set from the TargetFramework so I just want to ensure it is set correctly at this point.

**Review by weshaggard — APPROVED** [review]
One comment to verify otherwise looks good.

**Pull request merged**